aws/src/main/java/io/confluent/csid/config/provider/aws/SecretsManagerFactory.java

```diff
@@ -117,8 +117,9 @@
  */
 package io.confluent.csid.config.provider.aws;
 
-import com.amazonaws.services.secretsmanager.AWSSecretsManager;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
 
 interface SecretsManagerFactory {
-  AWSSecretsManager create(SecretsManagerConfigProviderConfig config);
+  SecretsManagerClient create(SecretsManagerConfigProviderConfig config);
+  void closeDefaultCredentials();
```

Comment:
Depending on what DefaultCredentialsProvider actually uses close for depending on the type; this was the only way I could expose the credentials to close correctly